terraform/modules/monitoring/variables.tf

```diff
@@ -33,3 +33,67 @@ variable "public_key_service_name" {
   type        = string
   description = "The Public Key Cloud Run service to monitor."
 }
+
+variable "jvs_ui_service_name" {
+  type        = string
+  description = "The JVS-UI Cloud Run service to monitor."
+}
+
+variable "notification_channel_email" {
+  type        = string
+  description = "The Email address where alert notifications send to."
+}
+
+variable "is_prod_env" {
+  type        = bool
+  description = "If deploying to a prod environment, true if is prod env, otherwise false."
+  default     = false
+}
+
+variable "cert_rotator_5xx_response_threshold" {
+  type        = number
+  description = "Send alert for Cert-Rotator when the number of response with 5xx code exceeds the threshold"
+  default     = 5
+}
+
+variable "cert_rotator_latency_threshold" {
```

Comment:
nit: add suffix `_ms` to the latency threshold

Reply:
Done